test/lib/simple.js

```diff
@@ -0,0 +1,48 @@
+const nock = require('nock')
```

Comment:
There are missing tests for generic error and status code != 200, we should aim to not decreasing coverage.

Reply:
Yes, I agree, and noticed that only after making the PR. I had previously looked at the tests and they seem a little bit complex to me that it wasn't obvious for how to slot it in or change it. 

Also, there probably should be a test for changing the URL and setting that incorrectly. 

Again, I'd appreciate it if you would just make the changes here since you are more facile with this than I am. Again, many thanks.

I was more concerned about the fact that armlet wasn't supporting the full as stated API. Also, I see that an issues from I guess the old armlet are missing in this repo: moving to using the "requests" library. I'll be adding that as an issue soon.